src/WebUI/dotnet/WebPortal/Views/Home/Index.cshtml

```diff
@@ -131,15 +143,53 @@ else
                     tr.append("<td>" + json.gpu_used + "</td>");
                     tr.append("<td>" + json.gpu_avaliable + "</td>");
                     tr.append("<td>" + json.AvaliableJobNum + "</td>");
-                    tr.append("<td>" + json.TotalJobNum + "</td>");
-
                     $('#cluster_status').append(tr);
                 })
             })
         }
+        function buildStorageTable() {
+            $.when.apply($, $.map(clusters, function (cluster) {
+                return $.ajax({
+                    url: '/api/dlws/GetMountPoints',
+                    data: { cluster: cluster },
+                    dataType: 'json',
+                    timeout: 10000
+                }).then(function (data) {
+                    return data;
+                }).catch(function () {
+                    return;
+                });
+            })).then(function () {
+                var cluster = null
+                var args = [].slice.call(arguments);
+                $.each(clusters, function (index) {
+                    if (args[index]) {
+                        clustersStorage[this] = args[index];
+                        if (cluster == null) {
+                            cluster = this.valueOf()
+                        }
+                    }
+                });
+                buildClusterTable();

```

Comment:
What's the reason of buildClusterTable again?

Reply:
This is obviously a typo.